tiledb/sm/query/readers/test/CMakeLists.txt

```diff
@@ -28,4 +28,5 @@ include(unit_test)
 
 commence(unit_test readers)
     this_target_sources(main.cc unit_reader_base.cc)
+    this_target_link_libraries(TILEDB_CORE_OBJECTS TILEDB_CORE_OBJECTS_ILIB)
```

Comment:
Try `this_target_object_libraries(baseline)`

If that doesn't work, then this should only be `TILEDB_CORE_OBJECTS_ILIB`

Reply:
That one works, thanks for suggestion!